bin/format

```diff
@@ -18,7 +18,7 @@ CLANG_FMT_STYLE_CFG = {
 
 IGNORE_PATTERNS = ["external"]
 
-DEFAULT_CLANG_FORMAT_VERSION=14
+DEFAULT_CLANG_FORMAT_VERSION=16
```

Comment:
Should it be 17?

Reply:
@mlxd tried 17 on at first, but decided on 16 in the end.